src/sentry/static/sentry/app/components/events/interfaces/spans/filter.tsx

```diff
@@ -0,0 +1,343 @@
+                  checkedQuantity > 0 && checkedQuantity !== operationNames.length
+                }
+                onClick={event => {
+                  event.stopPropagation();
+                  this.props.toggleAllOperationNameFilters(operationNames);
+                }}
+              />
+            </Header>
+            <List>
+              {operationNames.map((operationName: string) => {
+                const isActive = this.isOperationNameActive(operationName);
+
+                return (
+                  <ListItem key={operationName} isChecked={isActive}>
+                    <OperationDot backgroundColor={pickSpanBarColour(operationName)} />
+                    <OperationName>{operationName}</OperationName>
+                    <CheckboxFancy
+                      isChecked={isActive}
+                      onClick={event => {
+                        event.stopPropagation();
+                        this.props.toggleOperationNameFilter(operationName);
+                      }}
+                    />
+                  </ListItem>
+                );
+              })}
+            </List>
+          </MenuContent>
+        </DropdownControl>
+      </Wrapper>
+    );
+  }
+}
+
+const FilterLabel = styled('span')`
+  margin-left: ${space(1)};
+`;
+
+const Wrapper = styled('div')`
+  position: relative;
+  display: flex;
+
+  margin-right: ${space(1)};
+`;
+
+const StyledDropdownButton = styled(DropdownButton)<{hasDarkBorderBottomColor?: boolean}>`
+  white-space: nowrap;
+  max-width: 200px;
+
+  z-index: ${p => p.theme.zIndex.dropdown};
+
+  &:hover,
+  &:active {
+    ${p =>
+      !p.isOpen &&
+      p.hasDarkBorderBottomColor &&
+      `
+          border-bottom-color: ${p.theme.button.primary.border};
+        `}
+  }
+
+  ${p =>
+    !p.isOpen &&
+    p.hasDarkBorderBottomColor &&
+    `
+      border-bottom-color: ${p.theme.button.primary.border};
+    `}
+`;
+
+const MenuContent = styled('div')`
+  max-height: 250px;
+  overflow-y: auto;
+  border-top: 1px solid ${p => p.theme.gray400};
+`;
+
+const Header = styled('div')`
+  display: grid;
+  grid-template-columns: auto min-content;
+  grid-column-gap: ${space(1)};
+  align-items: center;
+
+  margin: 0;
+  background-color: ${p => p.theme.gray100};
+  color: ${p => p.theme.gray500};
+  font-weight: normal;
+  font-size: ${p => p.theme.fontSizeMedium};
+  padding: ${space(1)} ${space(2)};
+  border-bottom: 1px solid ${p => p.theme.borderDark};
+`;
+
+const List = styled('ul')`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const ListItem = styled('li')<{isChecked?: boolean}>`
+  display: grid;
+  grid-template-columns: max-content 1fr max-content;
+  grid-column-gap: ${space(1)};
+  align-items: center;
+  padding: ${space(1)} ${space(2)};
+  border-bottom: 1px solid ${p => p.theme.borderDark};
+  :hover {
+    background-color: ${p => p.theme.gray100};
+  }
+  ${CheckboxFancy} {
+    opacity: ${p => (p.isChecked ? 1 : 0.3)};
+  }
```

Comment:
Should this be built into CheckboxFancy instead of special casing here?

Reply:
I took this from the new breadcrumbs. It seems other places like the global header has the same styling. I'll follow up with a PR to consolidate these styles.